Attendance totals

cmkoller opened the pull request:
Add functionality for viewing total numbers of present/absent students to the Admin's Attendance page

Commit:
Add total numbers of present/absent students

Commit:
Mark acceptance criteria as complete

dpickett commented on spec/features/admin_views_attendence_spec.rb:
```diff
@@ -42,6 +45,36 @@
+
+    click_link 'View Attendance'
+
+    within(".columns.medium-6:first-child") do
```
this is probably a brittle selector - can we name the container with a class or ID to better isolate it and prevent this test from breaking if there are additional markup changes?

dpickett commented on spec/features/admin_views_attendence_spec.rb:
```diff
@@ -42,6 +45,36 @@
+  end
+
+  scenario "admin sees total number of active students" do
+    admin = FactoryGirl.create(:user, role: "admin")
```
I believe there is an `:admin` factory that should get this done for us. If not we should create one.

Commit:
Refactor tests
- Tests on attendance view now select columns based on class, to increase flexibility
- New "admin" factory allows better creation of admins in tests
